sql/hive-thriftserver/src/main/java/org/apache/hive/service/cli/thrift/ThriftCLIService.java

```diff
@@ -262,6 +262,28 @@ public TOpenSessionResp OpenSession(TOpenSessionReq req) throws TException {
     return resp;
   }
 
+  @Override
+  public TSetClientInfoResp SetClientInfo(TSetClientInfoReq req) throws TException {
+    // TODO: We don't do anything for now, just log this for debugging.
```

Comment:
Shall we make this as an IDed TODO like `TODO(SPARK-XXX)`?

Reply:
Actually this is copied from Hive: https://github.com/apache/hive/blob/1c5ce3f61e0dbf902cc7c75fb56f9f0b91a36e00/service/src/java/org/apache/hive/service/cli/thrift/ThriftCLIService.java#L347-L350

Reply:
Oh, got it. In that case, it looks fine.